devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/config-example/base/README.tpl.qute.md

```diff
@@ -0,0 +1,6 @@
+# {title}
+
+<p>This example displays mach speed in your favourite unit, depending on the specified Quarkus configuration.</p>
```

Comment:
This configuration could be moved to `codestart.yml` and reused in the `index.tpl.qute.html`

Reply:
@glefloch for now I can't, it contains `{config.file-name}` which should be evaluted.. I didn't split it in two and will use config when they will be evaluated: https://github.com/quarkusio/quarkus/issues/12792

Reply:
oh ok, I hadn't seen that one ;)